### PR TITLE
Reorganize account page

### PR DESCRIPTION
**Pull request by AlexAndBear**
## Description

image

## Related Issue

- Fixes https://github.com/owncloud/web/issues/11372

## How Has This Been Tested?

- test environment:
- test case 1:
- test case 2:
- ...

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests
- [ ] Documentation
- [ ] Maintenance (e.g. dependency updates or tooling)

## Open tasks:

- [ ] ...

### CODE REVIEW

**Comment by AlexAndBear** [comment]
e2e Tests are failing because of 🤡

**Comment by tbsbdr** [comment]
much more organised ⭐️⭐️⭐️⭐️⭐️
I suggest to limit the maximum width of the whole content to a fixed value like **80rem** so that following a row with the eyes on wide screens is easier.

**Comment by kulmann** [comment]
> much more organised ⭐️⭐️⭐️⭐️⭐️ 

Fully agree 😍 

> I suggest to limit the maximum width of the whole content to a fixed value like **80rem** so that following a row with the eyes on wide screens is easier.

Maybe we can choose the same value for the app store details page? There we also limited the width (although I don't know, yet, if it's too narrow).

**Comment by AlexAndBear** [comment]
@kulmann I tried that, was something about 600px, looked to narrow :(

**Comment by kulmann** [comment]
> @kulmann I tried that, was something about 600px, looked to narrow :(

I know it needs to be wider for the account page. Meant it the other way around, can we have a certain max width for the account page and then use the same for the app store details page? or is that too wide then? 😆

**Comment by AlexAndBear** [comment]
😵‍💫

**Comment by AlexAndBear** [comment]
ajdusted

**Comment by AlexAndBear** [comment]
Added responsive design. There is more work necessary e.G checkbox and label in same "row". But this will be happening in the follow-up

**Comment by tbsbdr** [comment]
mobile ❤️

- Fixes https://github.com/owncloud/web/issues/11453

File: packages/design-system/src/components/OcCheckbox/OcCheckbox.vue [hunk 1]
```diff
@@ -8,9 +8,10 @@
       :class="classes"
       :value="option"
       :disabled="disabled"
+      aria-label="label"
```
[the remark this answers is not in the document]
Comment (kulmann) [reply packages/design-system/src/components/OcCheckbox/OcCheckbox.vue:11]: The aria-label could/should even be omitted if `labelHidden` evaluates to `false`. 🤔 Reason: the visible label is sufficient regarding a11y, as long as it exists. But on the technical side that's only possible via `v-bind` with an object that sometimes contains `aria-label` and sometimes doesn't.

File: packages/web-runtime/src/components/Account/AccountTable.vue [hunk 2]
```diff
@@ -0,0 +1,95 @@
+<template>
+  <div class="account-table">
+    <slot name="header" :title="title">
+      <h2 class="account-table-title" v-text="title"></h2>
```
Comment (kulmann) [inline packages/web-runtime/src/components/Account/AccountTable.vue:4]: Please use a self-closing tag `<h2 />`.

**Review by kulmann — APPROVED** [review]
Love it so much! 😍

File: packages/design-system/src/components/OcCheckbox/OcCheckbox.vue (hunk 1 above)
[the remark this answers is not in the document]
Comment (AlexAndBear) [reply packages/design-system/src/components/OcCheckbox/OcCheckbox.vue:11]: Can you check with the current implementation ? setting the prop to null should omit aria-label entirely

**Review by JammingBen — APPROVED** [review]
💪

**Pull request merged**